src/WebInterface.hs

```diff
@@ -217,6 +210,11 @@ viewGroupedProjectQueues projects = do
     h2 "Recently integrated"
     mapM_ (uncurry $ viewList' viewPullRequestWithApproval) integrated
 
+  let awaitingApproval = reverse $ filterPrs (== Project.PrStatusAwaitingApproval)
```

Comment:
Small nit: I see almost the same computation duplicated. Maybe it could be refactored. Ranging from `awaitingApproval = reverse $ filterPrs (== project.PrStatusAwaitingApproval)` , defined somewhere in one place, to:
```haskell
displayAwaitingWith f = 
  let
    awaitingApproval = filterPrs (== project.PrStatusAwaitingApproval)
  in
    unless (null awaitingApproval) $ f awaitingApproval
```

Reply:
I tried to do this with @rudymatela but we found it less clear than simply repeating the pattern (for now), so we'll merge it without this. The filtering by state should probably be generalized anyway which seems out of scope for this PR, thanks for mentioning it